conda/common/configuration.py

```diff
@@ -1333,7 +1333,8 @@ def __call__(cls, *args, **kwargs):
         return self
 
 
-CONDARC_FILENAMES = (".condarc", "condarc")
+DEFAULT_CONDARC_FILENAME: Final = ".condarc"
+CONDARC_FILENAMES = (DEFAULT_CONDARC_FILENAME, "condarc")
```

Comment:
Since the non-`.` is also used (e.g., `sys_rc_path`) should we include that as well?

```suggestion
DEFAULT_CONDARC_FILENAME: Final = ".condarc"
ALTERNATIVE_CONDARC_FILENAME: Final = "condarc"
CONDARC_FILENAMES = (DEFAULT_CONDARC_FILENAME, ALTERNATIVE_CONDARC_FILENAME)
```